### PR TITLE
Add v 1.21 Comms team to release-team@ release-team-shadows@ and release-comms

### PR DESCRIPTION
**Pull request by divya-mohan0209**
- Added Comms shadows  to release-team@, release-team-shadows@, & release-comms@
- Deleted divya-mohan0209@gmail.com from release-team-shadows@
- Deleted Comms teams from previous cycles under release-comms@
- Added RT Lead, & RT Lead shadows to release-comms@

/assign @palnabarun @onlydole @kikisdeliveryservice @savitharaghunathan @bai

### CODE REVIEW

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

File: groups/sig-release/groups.yaml [hunk 1]
```diff
@@ -215,14 +215,17 @@ groups:
     managers:
       - tpepper@vmware.com # Release Team subproject owner
     members:
-      - keroden@microsoft.com
-      - killen.bob@gmail.com
```
Comment (kikisdeliveryservice) [inline groups/sig-release/groups.yaml:219]: Wondering why some of these were deleted?

**Review by kikisdeliveryservice — COMMENTED** [review]
Just a question

File: groups/sig-release/groups.yaml (hunk 1 above)
Comment (divya-mohan0209) [reply groups/sig-release/groups.yaml:219]: Per the description of the release-comms section "      External Release Communications list.
      Should be used when coordinating external release timelines e.g., with the CNCF.
      Membership should include SIG Chairs, Release Team Leads and Lead Shadows, and
      the Communications team for the current release cycle". They didn't fit it into any of the above (to the best of my knowledge), which is why I deleted them.

**Comment by palnabarun** [comment]
/lgtm
/approve

Thank you @divya-mohan0209 👍🏽

**Pull request merged**